src/app/contexts/RequestContext/getStatsDestination/index.js

```diff
@@ -1,12 +1,19 @@
 const getStatsDestination = ({ isUK = true, env = 'test', service }) => {
   let destination = '';
-
-  if (service === 'news') {
-    destination = isUK !== false ? 'NEWS_PS' : 'NEWS_GNL';
-  } else {
-    destination = 'WS_NEWS_LANGUAGES';
+  switch (service) {
+    case 'news':
+      destination = isUK !== false ? 'NEWS_PS' : 'NEWS_GNL';
```

Comment:
I think we could try something like
```
destination = isUK ? 'NEWS_PS' : 'NEWS_GNL';
```
maybe that would reduce it more

Reply:
That would reduce it, however it is like `isUK !== false ?` because this comparison also takes into consideration isUK being a null value. Whereas if it was just `isUK ?` it would only work with isUK being true or false but not with null, which it needs to be from my understanding.

Reply:
So are we saying if `isUK` is `null`, the user is from the UK, but if it is `false` they are not?  That seems suspect 🤔

Reply:
Yeah that's what I originally thought but when I looked at the Jest tests that were already in place when `isUK` is `null`, it implies that they are from the UK. Here's the file where the test are specifically line 55: [(Jest Tests)](https://github.com/bbc/simorgh/commit/3641405b1812aa8b57b4e00f2c55b483361749cb#diff-c02fdcc3ad00d36cba3f5f5f887c3af5).

@dr3 Don't suppose you would know why this is the case?

Reply:
I think our position was that if we don't know where the user is, we assume they are in the UK so we don't serve them ads.  Although I also thought this might have been superceeded by logic from upchain, which assumed unknown users were not in the UK.

Reply:
@HarveyPeachey fair enough - good job catching that - pretty much everyone would have just assumed that `null` meant they weren't in the UK 👍 

Probably worth raising an issue with core pod to look into fixing that - it's almost certainly going to cause bugs when requests from the UK have `isUK` set to `null`